sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -419,7 +420,6 @@ public int hashCode() {
     FLOAT,
     DOUBLE,
     STRING, // String.
-    DATETIME, // Date and time.
```

Comment:
I'm a little worried about this. Empirically many users are using schemas. Maybe we should start off by leaving DATETIME around and remove it later in another PR?

Reply:
Yeah we could deprecate it for a release or two before removing. Would we also need to keep support for the primitive DATETIME in IOs and SQL? I'd need to think about how that transition would work.